west/tools.py

```diff
@@ -0,0 +1,221 @@
+# Copyright (c) 2020 Espressif Systems (Shanghai) Co., Ltd.
```

Comment:
```suggestion
# Copyright (c) 2021 Espressif Systems (Shanghai) Co., Ltd.
```

Reply:
@shubhamkulkarni97 This file is not new. I have just renamed from `update.py` to `tool.py`. So I kept original date. Do you still think I need to update this?

Reply:
LGTM